ignite/version/version.go

```diff
@@ -100,7 +127,7 @@ func Long(ctx context.Context) string {
 
 	w.Init(b, 0, 8, 0, '\t', 0)
 
-	write("Ignite CLI version", Version)
+	write("Ignite CLI version", resolveDevVersion())
```

Comment:
You can pass the `ctx` here, it would be since since `resolveDevVersion()` involves a network call.

Reply:
https://github.com/ignite/cli/pull/3358/commits/2f9d66f5e0a29fd07bc58878786f90550fe699a0